Engine/World.cs

```diff
@@ -96,6 +129,19 @@ public void RemoveEntity(Entity entity)
             _entitiesRemovedBuffer.Add(entity);
         }
 
+        /// <summary>
+        /// Updates the region of an entity.
+        /// </summary>
+        /// <param name="entity">Entity to be updated</param>
+        public void UpdateEntityRegion(Entity entity)
```

Comment:
There's no update of the entity's region

Reply:
What do you mean?